src/lib.rs

```diff
@@ -45,10 +48,22 @@
 #[cfg(test)] #[macro_use] extern crate log;
 #[cfg(all(windows, test))] extern crate advapi32;
 
-use std::cmp::Ordering;
-use std::error::Error;
-use std::fmt;
-use std::ops::{Add, Sub};
+#[cfg(feature = "std")]
+mod std_impl;
+
+#[cfg(feature = "nightly")]
```

Comment:
Could this feature be dropped for now? I'd prefer to not have the crate all of a sudden require a nightly compiler when certain features are enabled on the crate